tests/operator_test.go

```diff
@@ -657,7 +657,14 @@ spec:
 
 		// make sure virt deployments use shasums again after each test
 		ensureShasums()
-	})
+    })
+
+    Describe("[testid:1746]should provide created and ready condition", func() {
+        kv := getCurrentKv()
+
+        By("Creating KubeVirt Object Created and Ready Condition")
```

Comment:
Nothing is created here. You just verify conditions on the existing KubeVirt CR (which is ok, no need to create a new one). And ready -> available again.